api/docs/release.dox

```diff
@@ -136,12 +136,16 @@ clients.
 The changes between version \DR_VERSION and 7.1.0 include the following minor
 compatibility changes:
 
- - Replaced NUM_SIMD_SLOTS with proc_num_simd_saved() and added the define
-   #MCXT_NUM_SIMD_SLOTS. Clients may set(DynamoRIO_NUM_SIMD_SLOTS_COMPATIBILITY ON)
-   in order to provide the define NUM_SIMD_SLOTS using proc_num_simd_saved().
-   The macro is not a constant expression and code relying on this needs to be
-   rewritten. DynamoRIO_NUM_SIMD_SLOTS_COMPATIBILITY is set automatically if
-   clients target version 7.1.0 or earlier.
+ - Added the function proc_num_simd_registers().
```

Comment:
This is not a compatibilty change: it should not be in this list.  It should be in the list below.

Reply:
Note moved.